src/Sylius/Component/Taxation/Model/TaxRate.php

```diff
@@ -40,6 +42,10 @@ class TaxRate implements TaxRateInterface
     /** @var string|null */
     protected $calculator;
 
+    protected ?DateTimeInterface $startDate = null;
```

Comment:
Why should I remove "use" and instead use "\" with DateTimeInterface?

Reply:
I don't have a perfect answer except that we are using DateTimeInterface in this way in Sylius and it would be consistent with other services. However, it is not a blocker for this PR of course 😃